tox.ini

```diff
@@ -13,8 +13,6 @@ deps=
 setenv =
     PYTHONPATH = {toxinidir}
     USERNAME = armi
-commands =
-    pytest --ignore=armi/utils/tests/test_gridGui.py armi
```

Comment:
@ntouran  This cuts the time that it takes to run the unit tests (with TOX) in half!